codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/AwsSignatureVersion4.java

```diff
@@ -120,7 +122,12 @@ private void writeNewV4SignerFunc(GoWriter writer, ServiceShape serviceShape) {
                         .type(SymbolUtils.createValueSymbolBuilder(SIGNER_INTERFACE_NAME).build())
                         .documentation("Signature Version 4 (SigV4) Signer")
                         .build())
-                .resolveFunction(SymbolUtils.createValueSymbolBuilder(SIGNER_RESOLVER).build())
+                .addconfigFieldResolver(
```

Comment:
minor nit should this be "add`C`onfigFieldResolver" ?  looks like the plural version is correct. `addConfigFieldResolvers`

Reply:
Yup this was a miss on my part!